Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -122,6 +163,28 @@
         "ServerCallFailed": "HTTP call to Jenkins server failed.",
         "ServerCallErrorCode": "HTTP response code: %d",
         "ArtifactSuccessfullyDownloaded": "Successfully downloaded artifact to %s",
-        "ArtifactDownloadUrl": "Downloading artifacts from: %s"
+        "ArtifactDownloadUrl": "Downloading artifacts from: %s",
+        "DownloadingCommitsAndWorkItems": "Downloading commits and work items",
+        "JenkinsDownloadingChangeFromCurrentBuild": "Cannot find start build, will be fetching commits and workitems from build used to download artifacts only: %s",
+        "DownloadingJenkinsChangeBetween": "Downloading commits and workitems between build %s to %s",
+        "GettingCommitsFromSingleBuild": "Downloading commits associated with the build %s",
+        "DownloadingContentFromJenkinsServer": "Downloading content form Jenkins server:%s with strict SSL:%s",
```

Comment:
typo - should be "from". Reword to "Downloading content from the Jenkins server ..."

Reply:
Fixed.